packages/playground/src/components/tf_notification.vue

```diff
@@ -1,77 +1,78 @@
-<template>
-  <div></div>
-</template>
-
 <script lang="ts" setup>
-import {
-  ContractStates,
-  type GqlContracts,
-  type GqlNodeContract,
-  type GqlRentContract,
-  GridClient,
-  NodeStatus,
-} from "@threefold/grid_client";
+import { ContractStates, type GqlContracts, type GqlNodeContract, type GqlRentContract } from "@threefold/grid_client";
+import type { GridNode } from "@threefold/gridproxy_client";
+import uniq from "lodash/fp/uniq.js";
 import { onMounted, ref } from "vue";
 
-import { getAllNodes } from "@/utils/get_nodes";
-
-import { useProfileManager } from "../stores";
+import { gridProxyClient } from "../clients";
+import { useGrid } from "../stores";
 import { createCustomToast, ToastType } from "../utils/custom_toast";
-import { getGrid } from "../utils/grid";
 import GracePeriodToast from "./mosha_toast/grace_period_toast.vue";
 import OfflineNodesToast from "./mosha_toast/offline_nodes_toast.vue";
 
-const profileManager = useProfileManager();
+const gridStore = useGrid();
 const contractsCount = ref(0);
 
-async function checkOfflineDeployments(grid: GridClient | null) {
-  const offlineNodesids = (await getAllNodes(grid, { flat: true, status: NodeStatus.down })) as number[];
-  const standByNodesids = (await getAllNodes(grid, { flat: true, status: NodeStatus.standBy })) as number[];
-  const offlineAndStandbyNodes = [...offlineNodesids, ...standByNodesids];
+async function checkOfflineDeployments() {
+  try {
+    const myContracts: GqlContracts = await gridStore.client.contracts.listMyContracts();
+    type Contract = GqlNodeContract | GqlRentContract;
+    const contracts: Contract[] = [...myContracts.nodeContracts, ...myContracts.rentContracts];
 
-  const myContracts: GqlContracts = await grid!.contracts.listMyContracts();
-  const contracts: (GqlNodeContract | GqlRentContract)[] = [...myContracts.nodeContracts, ...myContracts.rentContracts];
+    const ids = uniq(contracts.map(c => c.nodeID));
+    const settledNodes = await Promise.allSettled(ids.map(id => gridProxyClient.nodes.byId(id)));
 
-  const userOfflineDeployments = [];
-  const withPubIp = [];
+    const fakeNode = { status: "up" } as GridNode;
+    const offlineAndStandbyNodes = settledNodes
+      .map(n => (n.status === "fulfilled" ? n.value : fakeNode))
```

Comment:
Why do we set status:up here? Can you explain this, please?

Reply:
to filter it